docs/tutorials/python/ndarray.md

```diff
@@ -138,26 +140,27 @@ We can also dump a dict:
      [ 3.  3.  3.]]
  ```
 
-In addition, if we have set up distributed file systems, such as Amazon S3 and HDFS, we
-can directly save to and load from them. For example:
+In addition, if we have set up distributed file systems, such as Amazon S3 and
+HDFS, we can directly save to and load from them. For example:
 
  ```python
     >>> mx.nd.save('s3://mybucket/mydata.bin', [a,b])
     >>> mx.nd.save('hdfs///users/myname/mydata.bin', [a,b])
  ```
 
 ## Automatic Parallelization
-`NDArray` can automatically execute operations in parallel. This is desirable when you
-use multiple resources, such as CPU and GPU cards, and CPU-to-GPU memory bandwidth.
+A `NDArray` can automatically execute operations in parallel. This is desirable
+when you use multiple resources, such as CPU and GPU cards, and CPU-to-GPU
+memory bandwidth.
 
-For example, if we write `a += 1` followed by `b += 1`, and `a` is on a CPU card while
-`b` is on a GPU card, then we will want to execute them in parallel to improve the
-efficiency. Furthermore, data copies between CPU and GPU are expensive, so we
-want to run them in parallel with other computations.
+For example, if we write `a += 1` followed by `b += 1`, and `a` is on a CPU
+card while `b` is on a GPU card, then we will want to execute them in parallel
+to improve the efficiency. Furthermore, data copies between CPU and GPU are
+expensive, so we want to run them in parallel with other computations.
 
-However, finding statements that can be executed in parallel by eye is hard. In the
-following example, `a+=1` and `c*=3` can be executed in parallel, but `a+=1` and
-`b*=3` have to be sequentially executed.
+However, finding statements that can be executed in parallel by eye is hard. In
+the following example, `a+=1` and `c*=3` can be executed in parallel, but
+`a+=1` and `b*=3` have to be sequentially executed.
```

Comment:
let's change "have to be sequentially executed" to "must be executed sequentially"